quiz-app/src/components/app.js

```diff
@@ -0,0 +1,182 @@
+// import './App.css';
+import { TextField, Grid, FormControl, FormLabel, FormControlLabel, Button, Checkbox } from '@mui/material'
+import React, { useState } from "react"
+
+function App() {
+  const [question, setQuestion] = useState("");
+  const [choice1, setChoice1] = useState("");
+  const [choice2, setChoice2] = useState("");
+  const [choice3, setChoice3] = useState("");
+  const [choice4, setChoice4] = useState("");
+
+  const [answers, setAnswers] = React.useState({
+    answer1: false,
+    answer2: false,
+    answer3: false,
+    answer4: false
+  });
+
+  const { answer1, answer2, answer3, answer4 } = answers;
+
+  const handleAnswerCorrectnessChange = (event) => {
+    setAnswers({ ...answers, [event.target.name]: event.target.checked });
+  };
+
+
+  return (
+    <div className="Quiz_App" align="center">
+      <h1> Create your quiz </h1>
+      <Grid container direction={"column"} spacing={2} columns={{ xs: 12, sm: 12, md: 12, lg: 12, xl: 12 }}>
+        <Grid item xs={6} align="left">
+          <p1> Enter Question 1:</p1>
```

Comment:
You can also use typography.

Reply:
Indeed, thanks! Applied in ec56595892bd5163f597d5687c4dbbc8d20e6514